apps/medusa-be/src/admin/widgets/product-description-editor.css

```
@@ -0,0 +1,180 @@
+  --accentBase: #eff6ff;
+  --accentBg: #dbeafe;
+  --accentBgActive: #bfdbfe;
+  --accentBgHover: #dbeafe;
+  --accentBgSubtle: #eff6ff;
+  --accentBorder: #93c5fd;
+  --accentBorderHover: #60a5fa;
+  --accentSolid: #2563eb;
+  --accentSolidHover: #1d4ed8;
+  --accentText: #1d4ed8;
+  --accentTextContrast: #ffffff;
+  isolation: isolate;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)